ui/src/views/Tasks/TaskGroup/index.jsx

```diff
@@ -588,15 +587,12 @@ export default class TaskGroup extends Component {
     if (!error) {
       return true;
     }
-
     // Task groups do not necessarily have a decision task,
     // so handle task-not-found errors gracefully
-    return (
-      error.graphQLErrors &&
-      error.graphQLErrors[0] &&
-      error.graphQLErrors[0].statusCode === 404 &&
-      error.graphQLErrors[0].requestInfo.method === 'task'
-    );
+    const errorStatus = error.graphQLErrors.some((error, index) => {
+      return index == 0 && error.statusCode === 404 && error.requestInfo.method === 'task'
```

Comment:
Is there a reason why we are only checking for index 0?

Reply:
and really yet to know the reason for that

Reply:
I'm not sure I understand. Do you mind elaborating, please? :)

Reply:
so i noticed the ErrorPanel displays only the error at index 0, reasons that was done i don't really know yet. That is why i am factoring that in too

Reply:
The error panel displays the first error it gets if it receives an object, but if it's a string, it will display it as is.

The errors we get from graphql are not always in the same order. For example, it's possible that index 1 has status code 404 and requestInfo "task". We'll need to make sure we catch it here. The current logic only seems to look at index 0 which defeats the purpose of using `.some`.